assets/app/scripts/filters/util.js

```diff
@@ -405,4 +405,30 @@ angular.module('openshiftConsole')
       });
       return _.uniq(accessModes);
     };
+  })
+  .filter('truncate', function() {
+    return function(str, charLimit, useWordBoundary, newlineLimit) {
+      var truncated = str;
+
+      if (charLimit) {
+        truncated = truncated.substring(0, charLimit);
+      }
+
+      if (newlineLimit) {
+        var nthNewline = str.split("\n", newlineLimit).join("\n").length;
+        truncated = truncated.substring(0, nthNewline);
+      }
+
+      if (useWordBoundary !== false) {
```

Comment:
I suggest we flip the logic if we want to use word boundaries by default. Maybe

```
function(str, charLimit, ignoreWordBoundary, newlineLimit)
```

and

```
if (!ignoreWordBoundary) { ... }
```